addons/mail/static/src/models/composer_view.js

```diff
@@ -1367,6 +1346,10 @@ registerModel({
             inverse: 'composerViewOwner',
             isCausal: true,
         }),
+        compositions: many('MessageComposition'),
+        compositionsIndex: attr({
```

Comment:
this value should be global, not limited to the current composer view

Reply:
It's moved to messaging now